bundle/manifests/oran-o2ims.clusterserviceversion.yaml

```diff
@@ -1558,5 +1558,8 @@ spec:
   minKubeVersion: 1.28.0
   provider:
     name: Red Hat
+  relatedImages:
```

Comment:
Just remove this 'relatedImages' config; our midstream overlay will generate all the 'relatedImages' we need; we just need the env var renamed as you did a couple of lines above. Thanks !

Reply:
@rauhersu  This is automatically added by the bundle generation.